website/README.md

```diff
@@ -0,0 +1,28 @@
+# GeoTrellis Docs Site
+
+The GeoTrellis docs website is built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+## Developing on the docs
+
+Ensure you have Node 12+ installed.
+
+In one terminal, run `./sbt "project mdoc;mdoc --watch"` in the parent directory.
+This will run the mdoc watcher which recompiles docs in `../docs-mdoc` and writes the output to `./docs`.
+
+Start the docusaurus dev server with:
+
+```shell
+yarn start
```

Comment:
common knowledge but may be worth mentioning running 'yarn install' the first time?

Reply:
Good call, perhaps not for mostly Scala folks, updated.